test/functional/legacy/writefile_spec.lua

```diff
@@ -19,11 +19,11 @@ describe('writefile', function()
     execute('1 delete _')
 
     -- Assert buffer contents.
-    expect([[
+    expect([=[
       hello
       world!
       good
       morning
-      vimmers]])
+      vimmers]=])
```

Comment:
Those 2 changes are not needed, so I'd skip them (again, they're probably from `legacy2luatest.pl`... they don't matter, but we can just keep the old lines to cut down on the noise).